### PR TITLE
News abstraction

### PR DESCRIPTION
**Pull request by ddabble**
*Depends on #353, and should change base to `dev` after that PR has been merged.*

### Main features
* Made NewsBase abstract (e8f1a666585a8aeb70b1ecd3a84811a53708fba7)
  * **❗ Important note: ❗** The migration in this commit is fairly experimental; it has been tested multiple times locally (migrating both forward and backward), but not thoroughly. I personally think that it's most likely safe for deployment, though I'm not 100% sure, naturally.
* Refactored article and event list + detail pages (716f5cb8b75124c047be13e48e5955f7fa0e9a85)
* Refactored article and event edit pages (2fc33475f158195e22f139affbf840e56b3ca7d0)

### Additional highlights
* Improved tab-focusing links in header (a94966958c50087f042c8836d947f1ac3a3cc379)
* Added "Skip to main content" button for tab navigation (9b6747216c56c11ab4a2fc12f7bff2cf05fb902f)
* Added priority field to Secret (c51d8d48a247293f785a124c141ca967495eda81)
* Added `last_modified` field to various models (a8080a1e66946412b69ea4c5ff7ba022a8f19c7f)
* Improved multiple Django admin classes (79ec603c0ea2c64aa9f38c22ede3e1c32ba57352, 30ecdfef5a23a7b348b5fdf1abb8c40195e38ad0 and a8118b1f46111983bf5d5c9ffacabf41de534f9e)

### CODE REVIEW

File: internal/models.py [hunk 1]
```diff
@@ -201,14 +203,31 @@ def should_be_changed(self):
...
         max_length=100,
         unique=True,
         verbose_name=_("Title"),
     )
     content = MultiLingualRichTextUploadingField(verbose_name=_("Description"))
+    priority = models.IntegerField(
+        null=True,
+        blank=True,
+        verbose_name=_("Priority"),
+        help_text=_("If specified, the secrets are sorted ascending by this value."),
```
Comment (Fredrikaaa) [inline internal/models.py:228]: Skal denne teksten være synlig noe sted? Ser den ikke når jeg lager hemmeligheter.

Comment (ddabble) [reply internal/models.py:228]: Ah, det er sant! Det skal være fiksa nå 😊

File: faq/admin.py [hunk 2]
```diff
@@ -1,15 +1,40 @@
...
+from util import html_utils
+from util.admin_utils import link_to_admin_change_form, search_escaped_and_unescaped
 from web.multilingual.admin import MultiLingualFieldAdmin
 from .models import Category, Question
 
 
 class QuestionAdmin(MultiLingualFieldAdmin):
+    list_display = ('title', 'get_categries', 'last_modified')
+    list_filter = ('categories',)
+    search_fields = ('title', 'answer', 'categories__name')
```
Comment (Fredrikaaa) [inline faq/admin.py:13]: Skal det være to understreker her? 
Også get_categries på linje 11.

**Review by Fredrikaaa — APPROVED** [review]

File: faq/admin.py (hunk 2 above)
Comment (ddabble) [reply faq/admin.py:13]: Yeh, `categories__name` betyr at man søker på `name` feltet i `Category` modellen (gjennom `categories`-`ManyToMany`-feltet) 😄 

Bra at du fiksa opp `get_categries` 😊 (oppdaterte også et felt du hadde glemt, i 662c4e50c928f7991e385410f537619bc76026b7)

**Pull request merged**